File: Quantum-backend-apis/Quantum-backend-apis/DTO/WalkinDTO.cs
using System;
namespace Quantum_backend_apis.DTO
{
	public class WalkinDTO
	{
        public int job_id { get; set; }
        public string title { get; set; }
        public string general_ins { get; set; }
        public string instructions { get; set; }
        public string sys_req { get; set; }
        public string process { get; set; }
        public DateTime start_date { get; set; }
        public DateTime end_date { get; set; }
        public string location { get; set; }
        public string venue { get; set; }
        public List<int> roles { get; set; }
        public List<TimeOnly[]> timeslots { get; set; }
        public List<int> lpas { get; set; }
        public List<string> role_descs { get; set; }
        public List<string> requirements { get; set; }
    }
}

File: quantum-portal-angular/src/app/databaseServices/WalkinDTO.ts
export interface WalkinDTO {
    job_id: number;
    title: string;
    start_date: Date;
    end_date: Date;
    location: string;
    venue: string;
    roles: number[];
    timeslots: TimeOnly[][];
    lpas: number[];
    role_descs: string[];
    requirements: string[];
    general_ins: string,
    instructions: string,
    sys_req: string,
    process: string,
}

export interface TimeOnly {
    hour: number;
    minute: number;
    second: number;
    millisecond: number;
}
